cmd/utils.go

```diff
@@ -51,6 +52,17 @@ func addAutoApproveFlag(c *cobra.Command) *cobra.Command {
 	return c
 }
 
+func checkExists(cmd *cobra.Command, args []string) error {
+	path := args[0]
+	if path == "" {
```

Comment:
This seems like undesirable behavior:

```
ghpc deploy ""
Error: .ghpc/artifacts must be a writable directory
```

There is also an internal bug tracking being able to run "ghpc import inputs DOT" (i.e. the period character) where the deployment name is not properly inferred from the basename of the DOT directory. These may be related.

Reply:
Similarly if you are in a deployment directory, you get:

```
~/b/testfix❯❯❯ ghpc deploy ""
Testing if deployment group first requires adding or changing cloud infrastructure
Deployment group first requires adding or changing cloud infrastructure
Summary of proposed changes: Plan: 10 to add, 0 to change, 0 to destroy.
(D)isplay full proposed changes,
(A)pply proposed changes,
(S)top and exit,
(C)ontinue without applying
Please select an option [d,a,s,c]:
```

Reply:
Both of your examples are reproducible on `develop`. I would prefer to address it independently of this change. 

```sh
(develop %)$ ./ghpc deploy ""
Error: .ghpc/artifacts must be a writable directory
...
(develop %)$ ../ghpc deploy ""
Initializing deployment group primary
Testing if deployment group primary requires adding or changing cloud infrastructure
```